platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/cache/CookieCache.java

```diff
@@ -144,7 +145,9 @@ public void removeSamlAssertion(String key) {
     DataWrapper dataWrapper = cache.getIfPresent(key);
     if (dataWrapper != null) {
       LOGGER.debug(
-          "Expiring Saml assertion due to LogoutRequest\n[{}:{}]", key, dataWrapper.element);
+          "Expiring Saml assertion due to LogoutRequest\n[{}:{}]",
+          LogSanitizer.cleanAndEncode(key),
+          LogSanitizer.cleanAndEncode(dataWrapper.element.toString()));
```

Comment:
❓ Same here, and...

❗️ If this is stringifying XML documents or doing other DOM related things when we're not even at `debug` level...oof, that might be worth mitigating.